samples/languages/nodejs/nodejs-enclave_config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "args": [
+      "/usr/bin/node"
+      "app/cpubench.js"
```

Comment:
For clarity (since you're not setting cwd), make this `"/app/cpubench.js"`.

Reply:
Done